test/Staking.ts

```diff
@@ -199,5 +223,22 @@ describe("Staking contract", function () {
           accounts[3].address,
         ]);
     });
+
+    it("should be able to accept new validator after last one has unstaked", async () => {
+      // Reach full slot capacity
+      const value = ethers.utils.parseEther("1");
+      await Promise.all(accounts
+          .slice(0, 6)
+          .map((account) => {
+            stake(account, value);
+          })
+      );
+
+      // Account #0 unstakes
+      await contract.connect(accounts[0]).unstake();
+
+      // Account #6 should be able to become validator
```

Comment:
Typo here, `should be able to become a validator`

Reply:
Fixed 115588a7e5c6265e64aa2ce6755bf45a100913b9